app/models/email.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class Email < ApplicationRecord
+  class Association < ApplicationRecord
+    belongs_to :email
+    belongs_to :object, polymorphic: true
```

Comment:
Is this ever not a user?

Reply:
We can associate a single email to multiple objects in the database, depending on requirements. In this PR, some emails are associated to ParticipantProfiles as well - this can be useful if we want to check how many emails has been sent in regards to a particular ParticipantRecord. I believe there is already a card in JIRA to display `request_for_details` email status in admin panel (including `bounced/delivered`)

Reply:
I think that would only ever need to be for the `User` regarding whether their invite has been delivered or not? (sounds like the ticket I was working on)

Reply:
Schools has a contact email, and for some emails we might be more interested that we delivered them to a school than to a user

Reply:
Do we care? Is storing the email enough?

Reply:
@tonyheadford Email addresses on records can change, we need a persisted link between email and the reason why that email has been sent.